ContentProtection/OfflinePlayReadyAndWidevine/Program.cs

```diff
@@ -20,6 +19,8 @@ namespace OfflinePlayReadyAndWidevine
     class Program
     {
         private const string AdaptiveStreamingTransformName = "MyTransformWithAdaptiveStreamingPreset";
+        private const string TokenType = "Bearer";
+        private const string SourceUri = "https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/2b533311-b215-4409-80af-529c3e853622/Ignite-short.mp4";
```

Comment:
Not to make more work... but I realized that we are replicating this in all the samples now, and maybe I should have just placed it into the sample.env environment variables file in the first place.